orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -0,0 +1,315 @@
+ * and attempts to computationally infer them in each of Reactome's model organisms. Each RlE is broken down into its primary components (input, output, catalyst, and regulator), which are themselves broken
+ * into their PhysicalEntity subunits. The homology data used for the inference process currently comes from Ensembl Compara and is generated during the 'Orthopairs' step of the Reactome release process.
+ * After all inference attempts for each RlE has been completed in an organism, the pathways that contain the reactions are filled with these newly inferred ones. 
+ * 
+ *
+ */
+
+public class EventsInferrer
+{
+	static MySQLAdaptor dbAdaptor = null;
+	private static String releaseVersion = "";
+	private static GKInstance instanceEditInst;
+	private static GKInstance speciesInst;
+	private static Map<GKInstance,GKInstance> manualEventToNonHumanSource = new HashMap<GKInstance,GKInstance>();
+	private static List<GKInstance> manualHumanEvents = new ArrayList<GKInstance>();
+
+	@SuppressWarnings("unchecked")
+	public static void inferEvents(Properties props, String pathToConfig, String species) throws Exception
+	{
+		releaseVersion = props.getProperty("releaseNumber");
+		String pathToOrthopairs = props.getProperty("pathToOrthopairs");
+		String pathToSpeciesConfig = props.getProperty("pathToSpeciesConfig");
+		String dateOfRelease = props.getProperty("dateOfRelease");
+		int personId = Integer.valueOf(props.getProperty("personId"));
+		setReleaseDates(dateOfRelease);
+		
+		// Set up DB adaptor using config.properties file
+		String username = props.getProperty("username");
+		String password = props.getProperty("password");
+		String database = props.getProperty("database");
+		String host = props.getProperty("host");
+		int port = Integer.valueOf(props.getProperty("port"));
+		
+		dbAdaptor = new MySQLAdaptor(host, database, username, password, port);
+		setDbAdaptors(dbAdaptor);
+		
+		SkipInstanceChecker.getSkipList("normal_event_skip_list.txt");
+		
+		JSONParser parser = new JSONParser();
+		Object obj = parser.parse(new FileReader(pathToSpeciesConfig));
+		JSONObject jsonObject = (JSONObject) obj;
+
+		// Parse Species information (found in Species.json config file)
+		JSONObject speciesObject = (JSONObject) jsonObject.get(species);
+		JSONArray speciesNames = (JSONArray) speciesObject.get("name");
+		String speciesName = (String) speciesNames.get(0);
+		System.out.println("Beginning orthoinference of " + speciesName + ".");
+		JSONObject refDb = (JSONObject) speciesObject.get("refdb");
+		String refDbUrl = (String) refDb.get("url");
+		String refDbProteinUrl = (String) refDb.get("access");
+		String refDbGeneUrl = (String) refDb.get("ensg_access");
+		
+		// Creates two files that a) list reactions that are eligible for inference and b) those that are successfully inferred
+		String eligibleFilename = "eligible_" + species	+ "_75.txt";
```

Comment:
I'm curious about the significance of "75" in the filename...

Reply:
In the Perl version the threshold was 75 for inference to be considered valid. It was a variable that could be modified (though it never was). It could be removed and it remained there due to my Perl results mimicry